packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/connectors.html

```diff
@@ -0,0 +1,166 @@
+            data-sortable="asc"
+            data-type="string"
+            data-searchable
+            data-filterable
+        >
+            <div class="d-flex align-items-center">
+                <span
+                    class="round-icon oui-icon mr-2 oui-background-p-400 oui-color-p-050"
+                    ng-class="{
+                    'oui-icon-arrow-down': $row.connectorInformation.type === $ctrl.CONNECTOR_TYPES.SINK,
+                    'oui-icon-arrow-up': $row.connectorInformation.type === $ctrl.CONNECTOR_TYPES.SOURCE
+                    }"
+                >
+                </span>
+                <div>
+                    <h6
+                        ng-bind="$row.connectorInformation.name"
+                        class="mb-0"
+                    ></h6>
+                    <span class="oui-paragraph">
+                        <span
+                            >Version:
+                            {{$row.connectorInformation.version}}</span
+                        >
+                        <span>
+                            | Author: {{$row.connectorInformation.author}}</span
+                        >
+                        <span
+                            ng-if="$row.connectorInformation.documentationUrl"
+                        >
+                            |
+                            <a
+                                ng-href="{{$row.connectorInformation.documentationUrl}}"
+                                target="_blank"
+                                rel="noopener"
+                                >docs</a
+                            >
+                        </span>
+                    </span>
+                </div>
+            </div>
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_connectors_status' | translate"
+            data-property="status"
+            data-sortable="asc"
+            data-type="string"
+            data-searchable
+            data-filterable
+        >
+            <span
+                class="oui-badge"
+                data-ng-class="{
+                    'oui-badge_success': !$row.isPaused(),
+                    'oui-badge_warning':  $row.isPaused() ||  $row.isCreating()
+                }"
+            >
+                <span>{{$row.status}}</span>
+            </span>
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_connectors_tasks' | translate"
+            data-property="tasks"
+            data-sortable="asc"
+            data-type="string"
+        >
+            <div
+                ng-if="$row.getFailedTasks().length !== 0"
+                class="d-flex align-items-center oui-color-ae-500"
+            >
+                <span
+                    >{{$row.tasks.length -
+                    $row.getFailedTasks().length}}/{{$row.tasks.length}}</span
+                >
+                <span
```

Comment:
**issue (blocker)**:  Handling click events on the `span` will cause a accessibility issues. Use `<oui-button>`.

Reply:
Changed: https://github.com/ovh/manager/pull/6577/commits/ba63f0814b5ff1f65934c266b9af15886d28ffcf